textproc/libiconv/files/patch-c99.diff

```diff
@@ -6,7 +6,7 @@
  };
 -#ifdef __GNUC__
 +#if (defined (__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) || defined(__GNUC_STDC_INLINE__)
-+static inline
++inline
```

Comment:
This patch isn't needed anymore at all, and I already had the deletion staged, so I pushed it, which will create a conflict in this PR. You should just be able to drop the commit that modified this patch and then force-push to this branch.

Reply:
As suggested, I have dropped the fix part of the pull request since it has already been fixed.
All that remains is the minor tweaks that should not change the installed files.